ansible/group_vars/all/config.yml

```
@@ -798,4 +798,8 @@ configuration_all:
   PROMETHEUS_METRICS_COLLECT_METRICS_ROUTE_METRICS:
     server: true
     client: false
-    nuxtclient: false
\ No newline at end of file
+    nuxtclient: false
+  ROOT_URL_REDIRECT:
+    server: false
+    client: true
+    nuxtclient: false
```

set the value to "" as default to make sure that there is no value for it
```suggestion
  ROOT_URL_REDIRECT:
    value: ""
    server: false
    client: true
    nuxtclient: false
```

---

Line of code where it use at the legathy client https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/app.js#L208-L213